qemu/tests/cfg/rebase_second_snapshot_to_base.cfg

```diff
@@ -0,0 +1,47 @@
+- rebase_second_snapshot_to_base:
+    only raw
+    virt_test_type = qemu
+    type = rebase_raw_image_with_various_cache_mode
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    # md5sum binary path
+    md5sum_bin = "md5sum"
+    image_chain = "image1 sn1 sn2"
+    image_format_image1 = "raw"
+    image_name_sn1 = "images/sn1"
+    image_format_sn1 = qcow2
+    backing_chain_sn1 = yes
+    image_name_sn2 = "images/sn2"
+    image_format_sn2 = qcow2
+    backing_chain_sn2 = yes
+    # set size to "", so during snapshot creation
+    # the cmdline will not have specified size option
+    image_size_sn1 = ""
+    image_size_sn2 = ""
+    tmp_dir = /var/tmp
+    tmp_file_name = ${tmp_dir}/testfile
+    file_create_cmd = "dd if=/dev/urandom of=%s bs=1M count=512"
+    guest_file_name = ${tmp_file_name}
+    Windows:
+        guest_file_name = C:\testfile
+        x86_64:
+            sync_bin = WIN_UTILS:\Sync\sync64.exe /accepteula
+        i386, i686:
+            sync_bin = WIN_UTILS:\Sync\sync.exe /accepteula
+    variants:
```

Comment:
It would be better to split the cache mode and compatibility level to different variants parts.